src/context/schedule.tsx

```
@@ -34,26 +34,32 @@ function ScheduleProvider(props: ScheduleProviderProps): JSX.Element {
-      return result;
-    } catch (error) {
-      console.error('Error adding to schedule:', error);
-      throw error;
-    }
-  }, [flushPendingScheduleUpdates]);
+  const memoizedAddToSchedule = useCallback(
+    async (
+      dateTime: Date,
+      url: string,
+      room?: string
+    ): Promise<void | DocumentReference<DocumentData>> => {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)